_docs-sources/foundations/landing-zone/enable-control-tower.md

```diff
@@ -33,25 +37,37 @@ waiting on Control Tower Operations at the conclusion of the setup flow.
 ### Review pricing and select Regions
 
 1. Under **Region deny setting** select Enabled
+
    1. This ensures Control Tower policies and controls are unable to be bypassed by using a non-governed region
 
 1. Under **Select additional Regions for governance** select all regions where you plan to operate.
 
+   <details>
```

Comment:
This is a fairly clever solution. I think we might be able to find some sort of plugin, or even write a custom React component ourselves to create an expandable thumbnail, but let's roll with this as is for now!